config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,25 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-push-kubernetes-new-golang
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20200302-ca9526d
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/perf-tests=master,1189
```

Comment:
Change this line to master and remove the next one (--upload) once you're done with manual tests

Reply:
Done.